pandas/io/stata.py

```diff
@@ -1408,7 +1408,7 @@ def _read_old_header(self, first_char: bytes) -> None:
         self._time_stamp = self._get_time_stamp()
 
         # descriptors
-        if self._format_version > 108:
+        if self._format_version > 110:
```

Comment:
Can we switch this logic to >= and use only versions that have explicit support?

Reply:
Yes, I'll update it to >= 111.